package.json

```diff
@@ -33,7 +33,7 @@
     }
   },
   "scripts": {
-    "setup": "git submodule update --init --recursive && git submodule foreach --recursive git pull origin master && yarn --cwd ccd-definition-processor install",
+    "setup": "git submodule update --init --recursive && git submodule foreach git reset --hard 4d70e97 && yarn --cwd ccd-definition-processor install",
```

Comment:
are you sure this is what we want?

Reply:
Hi @yarpo yes we need this to allow the pipeline to pass. The newest version of the submodule has a breaking change, so this fixes the version to one that works for us. This submodule is not ours so we can't control versions and breaking changes.

Reply:
I'm not sure we should be fixing it, if we just remove `git submodule foreach --recursive git pull origin master` it'll use whatever reference is in the `ccd-definition-processor` file which already has a git reference

Reply:
cool - has been reverted now and we have the original sub-module commit in this branch